client/my-sites/email/email-management/home/email-list-inactive.tsx

```diff
@@ -31,6 +32,15 @@ const EmailListInactiveItem = ( { domain, source }: EmailListInactiveItemProps )
 					currentRoute,
 					source
 				) }
+				onClick={ () => {
+					sendNudge( {
+						nudge: 'email-comparison',
+						initialMessage: `I see you want to an email provider to your domain ${ domain.name }. I can give you a few tips on how to do that.`,
```

Comment:
We are missing a verb here, I assume this needs to be “want to add an email provider”?

Reply:
Thank you! I will fix it with the next changes I commit :)

Reply:
Thanks, @AllTerrainDeveloper.